src/cf.data.pre

```diff
@@ -1071,8 +1071,9 @@ DEFAULT: all src all
 DEFAULT: manager url_regex -i ^cache_object:// +i ^[^:]+://[^/]+/squid-internal-mgr/
 DEFAULT: localhost src 127.0.0.1/32 ::1
 DEFAULT: to_localhost dst 127.0.0.0/8 0.0.0.0/32 ::1/128 ::/128
+DEFAULT: to_linklocal dst 169.254.0.0/16 fe80::/10
```

Comment:
> @plygrnd: Suggest adding `fd00:ec2::254/128` for EC2.

That addition feels wrong to me because it is too specific to EC2 and because that address is not a link-local address.

We could add another ACL like `to_cloud_metaservice` that would match things like that, but I hope there is a more _general_ solution for the _default_ configuration than an ever-growing collection of company-specific addresses. Perhaps we can also deny access to all ULA addresses by default?

Reply:
An ACL definition of ```to_localnet``` using **dst** seems reasonable.

Reply:
> An ACL definition of `to_localnet` using **dst** seems reasonable.

I agree, but I am not going to add it in this PR for the following (combined) reasons:

1. Adding `to_localnet` is outside this PR scope. If we add `to_localnet`, then we should probably add `linklocal` as well. It can easily snowball into lots of unrelated, distracting changes.
2. Adding `to_localnet` will duplicate a lot of non-trivial RFC-number-sensitive `localnet` configuration lines while hiding that duplication from the casual observer because it happens in different places many lines apart.
3. I am strongly against defining such "standard" ACLs in squid.conf and allowing (and even encouraging!) the admin to "adapt" them (as we do today with the `localnet` ACL). Creating two ACLs with an identical set of addresses, each independently "adaptable" by the admin only exacerbates related problems!